app/models/alaveteli_pro/draft_info_request_batch.rb

```diff
@@ -17,8 +17,12 @@ class AlaveteliPro::DraftInfoRequestBatch < ActiveRecord::Base
   include AlaveteliPro::RequestSummaries
 
   belongs_to :user
-  has_and_belongs_to_many :public_bodies,
-     -> { reorder('public_bodies.name asc') }
+  has_and_belongs_to_many :public_bodies, -> {
+    I18n.with_locale(I18n.locale) do
```

Comment:
Explain why this is needed

- tests?
- commit message explaining why we need the change.